src/js/test/index.test.js

```diff
@@ -86,3 +86,144 @@ describe('currentScreenWidth', () => {
         expect(width).toBe(false);
     });
 });
+
+describe('withinBreakpoint', () => {
```

Comment:
It's always good to check that this method exists first before testing the implementation.

```
it('should exist', () => {
  expect(withinBreakpoint).toBeDefined()
})
```